lib/iris/fileformats/grib/__init__.py

```diff
@@ -911,9 +911,71 @@ def load_cubes(filenames, callback=None, auto_regularise=True):
     return iris.fileformats.rules.load_cubes(filenames, callback, grib_loader)
 
 
+def messages_from_filename(filename):
+    """
+    Return a generator of :class:`iris.fileformats.grib._message._GribMessage`
+    instances; one for each message in the supplied GRIB file.
+
+    Args:
+
+    * filename (string):
+        Name of the file to generate fields from.
+
+    """
+    return _GribMessage.messages_from_filename(filename)
+
+
+def load_pairs_from_fields(grib_messages):
+    """
```

Comment:
Just maybe, should this be called "load_pairs_from_messages", to match other names here ?